tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/EditorMicroservice.java

```diff
@@ -1220,14 +1228,28 @@ public Response exportApps(
                     .header("Siddhi-Docker-Key", dockerBuilderStatusKey)
                     .build();
         } catch (JsonSyntaxException e) {
-            log.error("Incorrect configuration format.", e);
+            log.error("Incorrect JSON configuration format.", e);
             return Response
                     .status(Response.Status.BAD_REQUEST)
+                    .entity("Incorrect JSON configuration format. " + e.getMessage())
+                    .build();
+        } catch (DockerGenerationException e) {
+            log.error("Failed docker generation. ", e);
```

Comment:
Rephrase as Exception caught while generating docker export artifacts. Update the k8 log as well

Reply:
Update response messages accordingly